File: anylang/c++/2469.温度转换.cpp
#include <iostream>
#include <vector>

class Solution {
   public:
    std::vector<double> convertTemperature(double celsius) {
        double Kelvin = celsius + 273.15;
        double Fahrenheit = celsius * 1.80 + 32.00;

        std::vector<double> rst = {Kelvin, Fahrenheit};
        return rst;
    }
};

int main() {
    Solution s;
    auto rst = s.convertTemperature(36.50);
    for (const auto& item : rst) {
        std::cout << item << std::endl;
    }

    return 0;
}

File: anylang/go/2469.温度转换.go
package main

import "fmt"

func convertTemperature(celsius float64) []float64 {
	var arr []float64
	arr[0] = celsius + 273.15
	arr[1] = celsius*1.80 + 32.00
	return arr
}

func main() {
	rst := convertTemperature(36.5)
    for( )
	fmt.Printf(rst)
}
